src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -100,7 +100,7 @@ public function block($messages, $type = null, $style = null, $prefix = ' ', $pa
      */
     public function title($message)
     {
-        $this->writeln(sprintf("\n<comment>%s</>\n<comment>%s</>\n", $message, str_repeat('=', strlen($message))));
+        $this->writeln(sprintf(PHP_EOL.'<comment>%s</>'.PHP_EOL.'<comment>%s</>'.PHP_EOL, $message, str_repeat('=', strlen($message))));
```

Comment:
I would like to split this line so that make it more readable
```php
$lines = array(
    sprintf('<comment>%s</>', $message),
    sprintf('<comment>%s</>', str_repeat('=', strlen($message))),
);

$this->writeln(implode(PHP_EOL, $lines).PHP_EOL);
```